packages/primitives/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Image } from 'theme-ui'
+import { SxStyleProp } from '@kodiak-ui/core'
+import { Box } from '../Box'
+
+export type AvatarProps = {
+  imageSrc?: string
+  text: string
+  sx?: SxStyleProp
+}
+
+export function Avatar({ imageSrc, text, sx = {} }: AvatarProps) {
```

Comment:
It could still be good to be able to pass in and default a variant and variantKey so that this can be customized from the theme.  What if we default in `variantKey = 'avatars' ` and pass through the variant to the image and box?